ship.config.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable import/no-commonjs */
+const fs = require('fs');
+const path = require('path');
+
+module.exports = {
+  mergeStrategy: { toSameBranch: ['master'] },
+  versionUpdated: ({ version, dir }) => {
+    fs.writeFileSync(
+      path.resolve(dir, 'src', 'lib', 'version.ts'),
+      `export default '${version}';\n`
+    );
+  },
+  beforeCommitChanges: ({ exec }) => {
+    exec('yarn doctoc');
+  },
+  pullRequestReviewer: ['@algolia/instantsearch-for-websites'],
+  slack: {
+    // disable slack notification for `prepared` and `releaseStart` lifecycle.
+    // Ship.js will send slack message only for `releaseSuccess`.
+    prepared: null,
+    releaseStart: null,
+    releaseSuccess: ({
+      appName,
+      version,
+      releaseTag,
+      latestCommitHash,
+      latestCommitUrl,
+      repoURL,
+    }) => ({
+      pretext: [
+        `:tada: Successfully released *${appName}@${version}*`,
+        '',
+        `:warning: Make sure you run \`yarn run release-templates\` in \`create-instantsearch-app\`.`,
```

Comment:
I'd remove the warning because it's stressful to see after having released haha

```suggestion
        `Make sure to run \`yarn run release-templates\` in \`create-instantsearch-app\`.`,
```